py-polars/tests/unit/test_lists.py

```diff
@@ -75,7 +75,7 @@ def test_dtype() -> None:
     a = pl.Series("a", [[1, 2, 3], [2, 5], [6, 7, 8, 9]])
     assert a.dtype == pl.List
     assert a.inner_dtype == pl.Int64
-    assert a.dtype.inner == pl.Int64  # type: ignore[attr-defined]
+    assert a.dtype.inner == pl.Int64  ### type: ignore[attr-defined]
```

Comment:
Quick drive-by comment: not sure why this change was introduced - please revert it. Same for line 269.

Reply:
It's failure in GitHub Action for it .
Here https://github.com/pola-rs/polars/commit/34ec4b4552b1baf5992920dc34dbf4bc109203dd

Reply:
Please consider look out these #6206
I did it that but mypy in GitHub Action stopped process , look out for commit please there

Reply:
Then you have to fix that specific mypy lint. The current "commenting out" is not how this issue should be addressed.

What is the mypy lint you got?

Reply:
@ritchie46 
Sorry not familiar with mypy much ,
Just executed pre commit on reach this issue .
How can I config mypy for this issue ?

Reply:
@ritchie46 [here](https://github.com/pola-rs/polars/actions/runs/3937755230/jobs/6735643260) is error